COMMUNICATIONS.md

```diff
@@ -0,0 +1,53 @@
+# OpenSearch Project Communication
+
+- [Overview](#overview)
+- [Slack](#slack)
+  - [Getting Started](#getting-started)
+  - [Workspace Channels](#workspace-channels)
+  - [Tips](#tips)
+
+## Overview
+
+The purpose of this document is to provide information regarding the communication channels for the OpenSearch Project. All communication is subject to the [OpenSearch Code of Conduct](CODE_OF_CONDUCT.md). Please see [CONTRIBUTING](CONTRIBUTING.md) if you're interested in contributing to the project.
+
+## Slack
+
+OpenSearch has a workspace on [Slack](https://opensearch.slack.com) to provide open communication channels for anyone interested in OpenSearch.
+
+The following guidelines include a Getting Started for steps to register and setup the workspace along with tips for using the workspace. Please read through these guidelines carefully and thoroughly.
+
+### Getting Started
+
+_Join Workspace_
+
+  * Register your account at https://opensearch.slack.com. You will be required to enroll in two-factor authentication to ensure the best possible security of your account.
```

Comment:
Given that people need the signup link, we should send people to https://opensearch.org/slack.html, where we'll be updating the signup link as it changes.

Reply:
A signup link is posted at https://opensearch.org/slack.html. You will be required to enroll in two-factor authentication to ensure the best possible security of your account.